src/Portfolio.sol

```diff
@@ -48,6 +52,8 @@ contract Portfolio is Ownable, SharedFund {
         assets[_symbol].isFlexible = _isFlexible;
         symbols.push(_symbol);
 
+        swap = new SwapToken(new ISwapRouter(0xE592427A0AEce92De3Edee1F18E0157C05861564));
```

Comment:
the swap router address should be provided as a constructor argument. Also, for which chain is the router address?